prow/jenkins/controller.go

```diff
@@ -222,9 +222,10 @@ func (c *Controller) Sync() error {
 
 	var reportErrs []error
 	if !c.skipReport {
-		reportTemplate := c.config().ReportTemplate
 		reportTypes := c.cfg().GitHubReporter.JobTypesToReport
+		jConfig := c.config()
 		for report := range reportCh {
+			reportTemplate := jConfig.ReportTemplateForRepo(report.Spec.Refs)
```

Comment:
Why does this look different between `plank` and `jenkins-operator`?

Reply:
Because `plank` is using `config config.Getter` but `jenkins-operator` has its own [method](https://github.com/kubernetes/test-infra/blob/master/prow/jenkins/controller.go#L113-L131) which it doesn't return a pointer. So if we just use `c.config().ReportTemplateForRepo(report.Spec.Refs)` we will get 
```
/controller.go:228:32: cannot call pointer method on c.config()
./controller.go:228:32: cannot take the address of c.config()
``` 
/shrug